src/_blog/libp2p-webrtc-browser-to-server.md

```diff
@@ -0,0 +1,223 @@
+---
+tags:
+- browser
+- transport
+- webrtc
+title: WebRTC in libp2p
+description:
+date:
+permalink: ""
+translationKey: ''
+header_image: /_.png
+author: David Di Maria
+---
+
+# WebRTC (Browser to Server) in libp2p
+
+<!--
+WebTransport in libp2p: Part 1 of Universal Browser Connectivity
+WebRTC (Browser to Server): Part 2 of Universal Browser Connectivity
+WebRTC (Browser to Browser): Part 3 of Universal Browser Connectivity
+-->
+This is the second entry in a series of posts on how libp2p achieves browser connectivity.
+Read about the [first article on WebTransport here](https://blog.libp2p.io/2022-12-19-libp2p-webtransport/).
+
+**Table of Contents**
+
+_IN DRAFT, NOT READY FOR REVIEW_
+
+[[toc]]
+
+## Overview
+
+The [libp2p project](https://libp2p.io) specifies many [transports](https://libp2p.io/implementations/#transports) developed in a variety of implementations (Rust, Go,  and JavaScript).  While connecting servers to servers has been around for a while via TCP and QUIC, there haven't been optimal solutions for connecting browsers to servers that work on a broad spectrum of browsers.  Additionally, there was a need to establish browser-to-browser connectivity in a decentralized way.
+
```

Comment:
```suggestion

Browser to server offerings, old and new, came with their own set of shortcomings.
This new libp2p WebRTC solution establishes browser to server connectivity in a decentralized way, across a broad spectrum of browsers, and in multiple libp2p implementations.

> Note: If you're familiar with the libp2p ecosystem, you may be wondering: Is this new? Hasn't there already been support for WebRTC in libp2p? The answer to both questions is yes. Although support has existed, this libp2p WebRTC solution is a fresh departure from older uses of the transport protocol in libp2p. We describe more further below.
```
This diff is for the line below "Other transports laid the groundwork for a WebRTC implementation in libp2p, but had some shortcomings, including centralized signaling and the need for a trusted TLS certificate, that created a need for a formal specification."

I think we should only allude to the deficiencies in older solutions in the overview. Specifics like "including centralized signaling and the need for a trusted TLS certificate, that created a need for a formal specification" can be expanded further below in the article